prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -261,6 +271,9 @@ class FHIRRouter(
             // ensure tracking is set
             actionHistory.trackCreatedReport(nextEvent, report)
 
+            // send event to Azure AppInsights
+            emitAzureEvent(report, message, null, bundle)
```

Comment:
Interesting, so this means if receiver is null the report didn't go anywhere. I think that's probably a fine way to communicate that.

Reply:
That was my thought. We could also omit sending this event entirely if we are only interested in truly "routed" reports. It would be easy enough to add a line in the query though to filter these ones out.